### ISSUE TITLE
Broken video playback if OpenGL is unavailable

### ISSUE THREAD

**Issue by City-busz** [Rafostar/clapper#497]
The commit 681da9e2f39fd5180317113e7349272586ae645d breaks the video playback if OpenGL is not supported by the system, because it tries to use `glsinkbin` unconditionally.

Please either remove this workaround, or ensure that GL context is available before using `glsinkbin`.

**Comment by Rafostar** [comment]
Indeed. `gst_element_factory_make` will return `glsinkbin` if its installed regardless if we can realize GL context. But GStreamer 1.24 have broken playback without using it.

We probably gonna need a sink to be a `bin` that hotplugs GL elements only if possible...
